<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/bg/pic/background.jpg);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>28</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard SE</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-background-color: rgba(255, 255, 255, 0);
alternate-background-color: rgba(255, 255, 255, 0);
gridline-color: rgba(255, 255, 255, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>137</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mode:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_mode">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard SE</family>
    </font>
   </property>
   <item>
    <property name="text">
     <string>单人模式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>双人模式</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AI蛇</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>人机大战</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(153, 102, 51, 160)</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(153, 102, 51, 160)</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>192</y>
     <width>46</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>274</y>
     <width>106</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>Special food:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>356</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Barrier:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cBox_Barrier_cross">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>356</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>crosswise</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cBox_bomb">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>274</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>bomb</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cBox_dimond">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>302</y>
     <width>66</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>dimond</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cBox_egg">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>274</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>egg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>191</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>r</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="snake_1_r">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>191</y>
     <width>42</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="snake_1_b">
   <property name="geometry">
    <rect>
     <x>318</x>
     <y>191</y>
     <width>42</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>222</y>
     <width>40</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>snake2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>191</y>
     <width>38</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>snake1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>191</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>b</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>222</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>g</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="snake_2_g">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>222</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>222</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>b</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="snake_2_b">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>222</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="snake_2_r">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>222</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>222</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>r</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>191</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>g</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="snake_1_g">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>191</y>
     <width>42</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cBox_Barrier_length">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>383</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>lengthwise</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cBox_moreFruit">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>302</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Chalkboard</family>
    </font>
   </property>
   <property name="text">
    <string>more fruit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
